Loop/Managers/LoopDataManager.swift

```diff
@@ -866,9 +866,14 @@ extension LoopDataManager {
             updateGroup.enter()
             carbStore.carbsOnBoard(at: now(), effectVelocities: settings.dynamicCarbAbsorptionEnabled ? insulinCounteractionEffects : nil) { (result) in
                 switch result {
-                case .failure:
-                    // Failure is expected when there is no carb data
-                    self.carbsOnBoard = nil
+                case .failure(let error):
+                    switch error {
+                    case .noData:
```

Comment:
Seems like DoseStore should be updated to not consider no carbs entered in the timeframe to be an "error".

Reply:
I'm for that. But then what is the `.noData` error for?

Reply:
oh... I see it is used for deleting and replacing.